src/main/java/com/developers/hireasenior/service/AuthService.java

```diff
@@ -26,7 +26,6 @@
 import com.developers.hireasenior.dto.response.ApiResponse;
 import com.developers.hireasenior.dto.response.RegistrationResponse;
 import com.developers.hireasenior.exception.EmailAlreadyExistsException;
-import com.developers.hireasenior.mapper.AccountMapper;
```

Comment:
I guess you accidentally removed this line in one of your commits, file gives error because it is not imported.

Reply:
fixed